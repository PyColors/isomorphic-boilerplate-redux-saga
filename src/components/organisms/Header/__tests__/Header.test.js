import renderer from 'react-test-renderer';
import React from 'react';
import header from '../Header';

describe('The Header <Component', () => {
  describe('The display element', () => {
    it('Should not regress', () => {
      const tree = renderer.create(<header />);

      expect(tree.toJSON()).toMatchSnapshot();
    });
  });
});
