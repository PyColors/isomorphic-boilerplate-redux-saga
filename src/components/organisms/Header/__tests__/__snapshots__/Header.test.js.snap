// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Header <Component The display element Should not regress 1`] = `<header />`;
